src/wallet/rpc/spend.cpp

```diff
@@ -1374,6 +1374,7 @@ RPCHelpMan sendall()
             RPCResult::Type::OBJ, "", "",
                 {
                     {RPCResult::Type::BOOL, "complete", "If the transaction has a complete set of signatures"},
+                    {RPCResult::Type::NUM, "weight", "The resulting transaction weight in bytes"},
```

Comment:
In "[rpc]: update `sendall` RPC to return the resulting tx weight" (8e256fe679236710549820ec48ac7c0793fcfd96):

"bytes" is not a unit of weight. I would expect that this is returning the weight in weight units or the vsize in vbytes.

Reply:
changes to "The resulting transaction weight in weight units"